hack/local-up-cluster.sh

```diff
@@ -1044,6 +1112,20 @@ if [[ "${KUBETEST_IN_DOCKER:-}" == "true" ]]; then
 
   # kubekins has a special directory for docker root
   DOCKER_ROOT="/docker-graph"
+
+  # to use docker installed containerd as kubelet container runtime
+  # we need to enable cri and install cni
+  # install cni for docker in docker
+  # containerd 1.4.12 installed by docker supports CNI version 0.4.0
+  CNI_VERSION=0.4.0
+  install_cni "${CNI_VERSION}"
```

Comment:
this should be optional to avoid conflicts with current installations, right?

Reply:
or just check that `CNI_CONFIG_DIR` is empty

Reply:
yeah, definitively I don't think is good idea downloading things by default,

Reply:
nevermind, this is correct, it already in 
`if [[ "${KUBETEST_IN_DOCKER:-}" == "true" ]]; then`

Reply:
this is only done for 
```
if [[ "${KUBETEST_IN_DOCKER:-}" == "true" ]];
```
which runs in the CI where containerd or any other cri not installed, so won't conflict with any current installtion